tests/python/rest_api/test_tasks.py

```diff
@@ -5631,21 +5631,23 @@ def test_can_import_datumaro_json(self, admin_user, tasks, dimension):
     @pytest.mark.parametrize(
         "format_name",
         [
-            "COCO 1.0",
-            "COCO Keypoints 1.0",
+            # FIXME: RejectionReason.detection_unsupported
+            # "COCO 1.0",
+            # "COCO Keypoints 1.0",
             "CVAT 1.1",
             "LabelMe 3.0",
             "MOT 1.1",
-            "MOTS PNG 1.0",
-            "PASCAL VOC 1.1",
+            "MOTS PNG 1.0",  # RejectionReason.unmet_requirements, but specific requirement information unavailable
+            # FIXME: looks like it desn't fail
+            # "PASCAL VOC 1.1",
             "Segmentation mask 1.1",
             "YOLO 1.1",
             "WiderFace 1.0",
             "VGGFace2 1.0",
-            "Market-1501 1.0",
+            "Market-1501 1.0",  # RejectionReason.unmet_requirements, but specific requirement information unavailable
```

Comment:
These comments are liable to become outdated. IMO, if you care _why_ the dataset is rejected, you should explicitly check it in the test. (And if not, then you shouldn't need the comments.)